tests/cc/sput.h

```diff
@@ -0,0 +1,272 @@
+/*
+ *  sput - Simple, Portable Unit Testing Framework for C/C++ v1.3.1
```

Comment:
interesting. I thought we're already using gtest, but since not, I think this one is preferred, since it's so tiny.